server/api.js

```diff
@@ -4,15 +4,140 @@ import pool from "./db";
 
 const router = Router();
 
+// Get all the tasks
 router.get("/tasks", (req, res) => {
 	pool.query("SELECT * FROM tasks").then((result) => {
-		// console.log(result.rows);
 		return res.send(result.rows);
 	});
 });
-
 router.get("/", (_, res) => {
-	return res.json({ message: "Hello, from the api route!" });
+	res.json({ message: "Hello, from the api route!" });
 });
```

Comment:
Please move this to the end of the routes as it might block other routes

Reply:
This deleted , after I push the change.

Reply:
It was there just for trial.